pyccel/ast/core.py

```diff
@@ -3924,8 +3924,21 @@ def rename(self, newname):
                               self.body, cls_name=self.cls_name)
 
 
-class F2PYFunctionDef(FunctionDef):
-    pass
+class BindCFunctionDef(FunctionDef):
+    def __new__(cls, *args, original_function, **kwargs):
+        return FunctionDef.__new__(cls, *args, **kwargs)
+
+    def __init__(self, *args, original_function, **kwargs):
+        self._original_function = original_function
+        return FunctionDef.__init__(self, *args, **kwargs)
```

Comment:
The `__init__` function does not return anything... I am surprised that Codacy does not complain about it.

Reply:
Functions that don't return anything actually return `None`. This is probably why it doesn't see it as a problem

Reply:
I know, but writing `return None` is definitely not idiomatic Python. `__init__` functions should never contain a return statement.

Reply:
By the way, could you please add a short docstring to this important class?